src/pages/header.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from "react";
+import { Transition } from "@headlessui/react";
+import Image from "next/image";
+
+import { headerSubSections, headerSubSectionsContentWallets } from "./data";
+
+
+export default function Header ({
+    content,
+    index,
+  }: {
+    content: any;
+    index: number;
+  }) {
+
+    const [isOpen, setIsOpen] = useState(false);
+
+
+    const handleDropDown = () => {
+        setIsOpen(!isOpen);
+    };
+
+    return (
+       
+     <div>
+        <div className="bg-white p-5 flex relative gap-2 font-normal space-x-10 text-sm items-center h-28 z-10">
+        <div className="gap-2 font-normal space-x-14 items-center flex">
+        <Image
+            src="/logo.png"
+            width={120}
+            height={120}
+            alt="test"
+            style={{ height: "100%", width: "auto" }}
+        />
+        <div className="dropdown relative w-max">
+            {headerSubSections.map((section, i) => (
+            <button
+                className="text-sm px-4 py-2.5 text-center inline-flex items-center hover:text-orange-600"
+                onClick={handleDropDown}
+                key={i}
+            >
+                {section.text}
+            </button>
+            ))}
+        </div>
+        </div>
+
+        <div className="gap-2 font-normal space-x-4 text-xs flex justify-end text-gray-500 p-2">
+        <a href="/#">Help</a>
+        <a href="/#">Find In-Store</a>
+        <span className="flex justify-items-center">
+            <Image
+            src="/email.png"
+            alt="mail"
+            width={20}
+            height={20}
+            style={{ height: "100%", width: "auto" }}
+            />
+        </span>
+        <span>
+            <Image
+            src="/magnifying-glass.png"
+            alt="mail"
+            width={20}
+            height={20}
+            style={{ height: "22px", width: "20px" }}
+            />
+        </span>
+        <span className="flex items-center">
+            <Image
+            src="/trolley.png"
+            alt="mail"
+            width={22}
+            height={20}
+            style={{ height: "22px", width: "20px" }}
+            />
+        </span>
```

Comment:
Can these be mapped over instead? Cleaner implementation

Reply:
completed inside of feature/product-card branch